gtfs/spec/en/reference.md

```diff
@@ -123,7 +123,7 @@ File: **Required**
 
 |  Field Name | Type | Required | Description |
 |  ------ | ------ | ------ | ------ |
-|  `stop_id` | ID | **Required** | Identifies a stop, station, or station entrance. The term "station entrance" refers to both station entrances and station exits. Stops, stations or station entrances are collectively referred to as locations. Multiple routes may use the same stop. |
+|  `stop_id` | ID | **Required** | Identifies a stop, station, or station entrance. `stop_id` is an internal ID, not intended to be shown to riders. <br><br> The term "station entrance" refers to both station entrances and station exits. Stops, stations or station entrances are collectively referred to as locations. Multiple routes may use the same stop. |
```

Comment:
Is the <br><br> allowed in mark down? I would expect two spaces.

Reply:
The \<br>\<br> tag is allowed in markdown. It appears like <br><br> this.